interfaces/builtin/pulseaudio_test.go

```diff
@@ -64,3 +75,42 @@ func (s *PulseAudioInterfaceSuite) TestSanitizePlug(c *C) {
 	err := s.iface.SanitizePlug(s.plug)
 	c.Assert(err, IsNil)
 }
+
+func (s *PulseAudioInterfaceSuite) TestSecCompOnClassic(c *C) {
+	seccompSpec := &seccomp.Specification{}
```

Comment:
Same question as for the MIR interface, why is this called on classic and we do not mock on classic here?